docs/03-01-service-description.md

```diff
@@ -1,6 +1,6 @@
 # Service description
 
-Kyma Environment Broker (KEB) is compatible with the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBA) specification. It provides a ServiceClass that provisions Kyma Runtime on a cluster.
+Kyma Environment Broker (KEB) is compatible with the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI) specification. It provides a ServiceClass that provisions Kyma Runtime on a cluster.
```

Comment:
It's better to use "SAP BTP, Kyma runtime", or the abbreviated name "SKR" for the managed offering and just "Kyma" for the open-source project. I suggest replacing all "Kyma Runtime"  occurrences with "SKR" after introducing the full name at the beginning. The abbreviation is already used in some sections of this doc.

Reply:
suggestion applied.